### PR TITLE
[Filter] Fix CMake Build Bug - @open sesame 09/04 17:44

### PR DESCRIPTION
**Pull request by jijoongmoon**
When DISABLE_TENSORFLOW_LITE=ON or DISABLE_TENSORFLOW=ON, it is not
working becasue of the wrong CMakeList.txt.

**Changes proposed in this PR:**
- Fix IF-ENDIF statement in CMakeList.txt

**Self evaluation:**
1. Build test:	 [X]Passed [ ]Failed [ ]Skipped
2. Run test:	 [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: jijoong.moon <jijoong.moon@samsung.com>

---
# [Template] PR Description

In general, github system automatically copies your commit message for your convenience.
Please remove unused part of the template after writing your own PR description with this template.
```bash
$ git commit -s filename1 filename2 ... [enter]

Summarize changes in around 50 characters or less

More detailed explanatory text, if necessary. Wrap it to about 72
characters or so. In some contexts, the first line is treated as the
subject of the commit and the rest of the text as the body. The
blank line separating the summary from the body is critical;
various tools like `log`, `shortlog` and `rebase` can get confused 
if you run the two together.

Further paragraphs come after blank lines.

**Changes proposed in this PR:**
- Bullet points are okay, too
- Typically a hyphen or asterisk is used for the bullet, preceded
  by a single space, with blank lines in between, but conventions vary here.

Resolves: #123
See also: #456, #789

**Self evaluation:**
1. Build test: [ ]Passed [ ]Failed [*]Skipped
2. Run test: [ ]Passed [ ]Failed [* ]Skipped

**How to evaluate:**
1. Describe how to evaluate in order to be reproduced by reviewer(s).

Add signed-off message automatically by running **$git commit -s ...** command.

$ git push origin <your_branch_name>
```

### CODE REVIEW

**Comment by myungjoo** [comment]
Uh.. is this really necessary?

With the following code, it appears to work property without this commit:
```
IF(DISABLE_TENSORFLOW_LITE)
MESSAGE("Disable Tensorflow Lite")
ENDIF(DISABLE_TENSORFLOW_LITE)
IF(NOT DISABLE_TENSORFLOW_LITE)
MESSAGE("Not disable Tensorflow Lite")
ENDIF(NOT DISABLE_TENSORFLOW_LITE)
```

The problem might be caused by something else..?

**Comment by myungjoo** [comment]
Note that ```DISABLE_TENSORFLOW_LITE``` is not an environment variable, it's a CMAKE option:
```
$ cmake .. -DDISABLE_TENSORFLOW_LITE=ON
```

File: gst/tensor_filter/CMakeLists.txt [hunk 1]
```diff
@@ -5,21 +5,21 @@ set(FILTER_SOURCE tensor_filter.c tensor_filter_custom.c)
 set(FILTER_LIB tensor_filter)
 set(FILTER_TARGET tensor_filter)
 
-IF(NOT DISABLE_TENSORLOW_LITE)
+IF(NOT ${DISABLE_TENSORFLOW_LITE})
```
Comment (leemgs) [inline gst/tensor_filter/CMakeLists.txt]: Nice catch.

File: gst/tensor_filter/CMakeLists.txt [hunk 2]
```diff
@@ -5,21 +5,21 @@ set(FILTER_SOURCE tensor_filter.c tensor_filter_custom.c)
...
   list(APPEND FILTER_SOURCE tensor_filter_tensorflow_lite.c)
   list(APPEND FILTER_LIB tensor_filter_tflitecore)
   list(APPEND FILTER_TARGET tensor_filter_tflitecore)
   PKG_CHECK_MODULES(TENSORFLOW_LITE REQUIRED tensorflow-lite)
   LINK_DIRECTORIES(${TENSORFLOW_LITE_LIBRARY_DIRS})
-ENDIF(NOT DISABLE_TENSORLOW_LITE)
+ENDIF(NOT ${DISABLE_TENSORFLOW_LITE})
```
Comment (leemgs) [inline gst/tensor_filter/CMakeLists.txt]: It looks better.

**Pull request merged**